pkg/criu/criu.go

```diff
@@ -21,3 +26,16 @@ func CheckForCriu(version int) bool {
 	}
 	return result
 }
+
+func MemTrack() bool {
+	features, err := criu.MakeCriu().FeatureCheck(
+		&rpc.CriuFeatures{
+			MemTrack: proto.Bool(true),
+		},
+	)
+	if err != nil {
+		return false
+	}
+
+	return *features.MemTrack
```

Comment:
Can you do a nil-check on this before returning? A null return may be unlikely, but not segfaulting is important.

Reply:
done